mod/friendica.php

```diff
@@ -75,7 +75,8 @@ function friendica_content(App $a)
 {
 	$o = '<h1>Friendica</h1>' . PHP_EOL;
 	$o .= '<p>';
-	$o .= L10n::t('This is Friendica, version') . ' <strong>' . FRIENDICA_VERSION . '</strong> ';
+	$o .= L10n::t('This is Friendica, version') . ' <strong>' . FRIENDICA_VERSION .
```

Comment:
Maybe you should add the "Postupdate Version" in a new line?

The Friendica version, the one that is listed everywhere is without that postupdate number. It is not published in the nodeinfo, and except for the purpose of checking when to run the DB update it is not needed - if I remember correctly.